scipy/meson.build

```diff
@@ -184,6 +208,10 @@ elif lapack_name == 'openblas'
 else
   lapack = dependency(lapack_name)
 endif
+lapack_dep = declare_dependency(
+  dependencies: lapack,
+  compile_args: _args_blas_lapack
```

Comment:
It's a little concerning that some targets need `link_with: blas_lapack_wrapper_lib` and some don't. It would probably be nicer to include that `link_with` here, then using `lapack_dep`/`blas_dep` anywhere else will be robust.

Reply:
Another really good catch! I checked `scipy.linalg._interpolative`, one of the targets where this was missing, and realized that the only reason nothing broke was because Accelerate provides the updated `$NEWLAPACK`-suffixed symbols alongside the suffix-less old symbols. To prevent any wrapper omissions going forward, I added `link_with: [g77_abi_wrappers, blas_lapack_wrapper_lib]` to `lapack_dep`/`blas_dep`.